wholesym/src/config.rs

```diff
@@ -20,6 +33,7 @@ pub struct SymbolManagerConfig {
     pub(crate) use_spotlight: bool,
     pub(crate) debuginfod_cache_dir_if_not_installed: Option<PathBuf>,
     pub(crate) debuginfod_servers: Vec<(String, PathBuf)>,
+    pub(crate) precog_data: Option<PrecogDataContainer>,
```

Comment:
As I said in https://github.com/Unity-Technologies/usamply/pull/1#discussion_r1601910497, I would prefer storing it on the helper rather than in the config, and without an extra wrapper struct.

Reply:
Oh, I misunderstood that comment! I missed the part about keeping it in the helper. Will do.

Reply:
(The wrapper struct was to allow for Debug FWIW -- `SymbolMapTrait` does not imply Debug, so I would have had to define another trait and make sure all symbol map objects where `Debug`, or implement debug manually for `SymbolManagerConfig`)